modules/fluid_properties/unit/src/FlibeFluidPropertiesTest.C

```diff
@@ -106,6 +106,9 @@ TEST_F(FlibeFluidPropertiesTest, specificInternalEnergy)
 
   ABS_TEST(_fp->e_from_p_T(p, T), 1.9326997921944547e+06, REL_TOL_SAVED_VALUE);
   DERIV_TEST(_fp->e_from_p_T, p, T, REL_TOL_DERIVATIVE);
+
+  ABS_TEST(
+      _fp->e_from_p_rho(p, _fp->rho_from_p_T(p, T)), 1.9326997921944547e+06, REL_TOL_SAVED_VALUE);
```

Comment:
This same value is used above. Just define it once (`const Real e = 1.9326997921944547e+06`)